content/en/community/code-of-conduct.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Kubernetes Community Code of Conduct
+body_class: code-of-conduct
+---
+
+<a class="td-offset-anchor"></a>
+<section class="row td-box td-box--1 position-relative td-box--gradient td-box--height-auto">
+    <div class="container text-center">
+        <span class="h4 mb-0">
+            <h1>Kubernetes Community Code of Conduct</h1>
+            <p>
+            Kubernetes follows the
+            <a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Code of Conduct</a>.
+            The text of the CNCF CoC is replicated below, as of
+            <a href="https://github.com/cncf/foundation/blob/71412bb029090d42ecbeadb39374a337bfb48a9c/code-of-conduct.md">commit 71412bb02</a>.
```

Comment:
I updated the link to the most recent commit.
```suggestion
            <a href="https://github.com/cncf/foundation/blob/fee478bdcd72a706662de28b4666bd7a573d68c6/code-of-conduct.md">commit fee478bdc</a>.
```

Reply:
This PR is about a theme change. If you want to update the code of conduct, would you consider doing that in a separate PR?

Reply:
Oh, yes. I’ll like to open an issue first. I wish we could refer to the code of conduct page rather than a particular commit hash. Manually updating the hash every time feels repetitive.